multi-agent/lib/mas/blueprints/exceptions.py

```
@@ -2,6 +2,13 @@
 Custom exceptions for Blueprint operations.
 Provides specific error types for better debugging and error handling.
 """
+import logging
+import re
+from typing import Any, Dict, List
```

Ruff (0.15.20)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)